spec/pureComputedBehaviors.js

```diff
@@ -5,6 +5,11 @@ describe('Pure Computed', function() {
         expect(ko.isComputed(computed)).toEqual(true);
     });
 
+    it('Should advertise that instances are pure computed', function () {
```

Comment:
I think there should also IMHO be assertions that the `ko.isPureComputed` fails for regular computeds and observables.